src/codes/clj/contest/submission_runner/wire/in/submission.clj

```diff
@@ -0,0 +1,28 @@
+(ns codes.clj.contest.submission-runner.wire.in.submission)
+
+(def Submission
+  [:map
+   [:code string?]
+   [:language [:enum :clojure]]
+   [:test-cases [:map-of :keyword
+                 [:map
+                  [:input :any]
+                  [:output :any]]]]])
+
+(comment
+  (require '[malli.generator :as m])
+  (require '[malli.core :as m.core])
+  (m/generate Submission)
+  (m.core/validate
+   Submission
+   {:code "(ns runner 
+             (:require [clojure.test :refer [use-fixtures]])
+             (defn my-sum [a b] (+ a b)))"
+    :language :clojure
+    :test-cases {:case-1 {:input "(my-sum 1 2)"
+                          :output 3}
+                 :case-2 {:input "(my-sum 2 3)"
+                          :output 5}}})
```

Comment:
@rafaeldelboni what do you think of this input?

Reply:
this way I will have to merge it, so I think is better to receive it already merged,

Reply:
Who is going to send this payload the frontend?

Reply:
The idea is use this code to build the code that will run in the pod right?

Reply:
> Who is going to send this payload the frontend?
Hm, 🤔 idk why I was thinking of having two backends.

Reply:
> The idea is use this code + test cases to build the code that will run in the pod right?
Yep, the idea was to merge it in some way and send to the pod

Reply:
nice 👍🏽